activities.json

```diff
@@ -443,6 +443,18 @@
     "title": "Reporting API",
     "url": "https://w3c.github.io/reporting"
   },
+  {
+    "ciuName": null,
+    "description": "This document specifies an API that allows web applications to request a screen wake lock. Under the right conditions, and if allowed, the screen wake lock prevents the system from turning off a device's screen.",
+    "id": "screen-wake-lock",
+    "mozBugUrl": null,
+    "mozPosition": "worth prototyping",
+    "mozPositionDetail": "As the scope of the specification has been reduced to screen wake locks, it's worth prototyping this API in a manner that restricts it to foreground first-party content. Additionally, the API appears flexible enough that a permission grant can be performed asynchronously, allowing us to evaluate the most appropriate permission model should we choose to ship this API in the future. As the API allows the capability to be revoked at any time, we can also prototype eagerly granting, notifying the user what's going on, and allowing users to disable the capability entirely - either per origin, or globally through a browser setting.",
```

Comment:
Maybe worth mentioning somewhere that the potential downside is the risk of abuse by sites that don't have a strong need for it, degrading battery life for the sake of engagement?  Or something like that?